internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_providers.go

```diff
@@ -14,9 +14,11 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 )
 
+const WORKFORCE = "WORKFORCE"
```

Comment:
i would favor to have that const in the model file as it's being used in multiple files and that seems to be the most "common" one

Reply:
makes sense, done